### ISSUE TITLE
Titanium error with "module" and "app.js" named modules

### ISSUE THREAD

**Issue by tonylukasavage** [tonylukasavage/ti-commonjs#8]
In `Resources/app.js` the `module` object and all associated CommonJS free variables are undefined. Aside from the obvious problems this raises, it also blocks #7. This is trivially reproducible:
### app.js

``` js
module.exports = 123; // error
exports.foo = 'bar'; // error
```

Additionally, this error occurs in _any_ module named `app.js`. So even if it's not the main app.js entry point, it still suffers from this problem. So you would have the same exact problems as above with `Resources/someFolder/app.js`.

**Comment by tonylukasavage** [comment]
This needs to be logged as a Titanium SDK bug.

**Comment by tonylukasavage** [comment]
resolved in 8b9fe77f64b6aaf16e8836c004bd0bc4a43a5c62
